src/extension.ts

```diff
@@ -20,14 +20,17 @@ export function activate(context: vscode.ExtensionContext) {
 					if (lastSelectionTime != thisSelectionTime) return;
 
 					if (config.autoHideSideBar) {
-						//vscode.commands.executeCommand("workbench.files.action.focusFilesExplorer");
-						vscode.commands.executeCommand("workbench.extensions.action.showInstalledExtensions");
+                        //vscode.commands.executeCommand("workbench.files.action.focusFilesExplorer");
```

Comment:
I think that was an incidental update. It should be resolved by the second commit -- 93f6e95